tests/test/Invalid.t.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.5.2;
+
+contract OldCompilerTest {
+
+    function testAssert(uint x) public {
```

Comment:
if we compile this test with a recent solc, it won't actually exercise the `invalid` opcode path, right? might be worth it to have a test where we hardcode an `invalid` opcode